exif-reader.d.ts

```diff
@@ -38,11 +84,33 @@ interface XmpTags {
     [name: string]: XmpTag
 }
 
+interface ThumbnailTags {
```

Comment:
Noticed something while checking it out - using [this exif image example (zip)](https://github.com/mattiasw/ExifReader/files/4970604/88368217-e1c3a680-cd8d-11ea-83b6-aa5821ec5f81.zip) from this expected type I get this object: 
```json
{
  "type": "image/jpeg",
  "image": {},
  "base64": "base 64 stuff",
  "Compression": "...",
  "XResolution": "...",
  "YResolution": "...",
  "ResolutionUnit": "...",
  "JPEGInterchangeFormat": "...",
  "JPEGInterchangeFormatLength": "...",

  "Orientation": "...",
  "YCbCrPositioning": "..."
}
```

the last two seems to be missing. thanks again!

Reply:
Nice, I also discovered some more and added them all. The PR is updated.